fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -127,6 +132,13 @@
     private static final com.ibm.fhir.model.type.String SC_NOT_FOUND_STRING = string(Integer.toString(SC_NOT_FOUND));
     private static final com.ibm.fhir.model.type.String SC_OK_STRING = string(Integer.toString(SC_OK));
     private static final String TOO_MANY_INCLUDE_RESOURCES = "Number of returned 'include' resources exceeds allowable limit of " + SearchConstants.MAX_PAGE_SIZE;
+    private static final ZoneId UTC = ZoneId.of("UTC");
+
+    // default number of entries in system history if no _count is given
+    private static final int DEFAULT_HISTORY_ENTRIES = 100;
+
+    // clamp the number of entries in system history to 1000
+    private static final int MAX_HISTORY_ENTRIES = 1000;
```

Comment:
seems this aligns to Search Page Size

Reply:
Yes, but different because we are only including the urls, not the actual resources. We keep the defaults fairly low because it's likely a client will turn around and request everything in a single call. A more sophisticated client (wanting better performance) will distribute the request to a thread-pool and issue the GETs concurrently. Need to add this to the users' guide.